recipes/supervisor/post-link.sh

```diff
@@ -0,0 +1,6 @@
+if [[ $(id -u) == 0 ]]; then
+    mkdir -p /etc/supervisord/ && ln -s $PREFIX/etc/supervisord/ /etc/supervisord/
+    ln -s $PREFIX/etc/supervisord.conf /etc/supervisord.conf
```

Comment:
This is kind of uncharted territory actually.  It's still a question for me what the "best practice" should be for conda doing OS management tasks like directory creation (outside of $PREFIX), user creation, setting permissions, etc.  The native OS package managers do all of this when they install packages; conda should too.  At some point.

This part is just a little nudge in that direction.  If you've already installed conda as root, then go ahead and link these files into the `/` file structure.